spec/once_spec.rb

```diff
@@ -11,83 +11,118 @@
   let(:thing_to_execute)  { double(execute: "foo") }
   let(:params) {{foo: "bar"}}
 
-  it "executes the given command" do
```

Comment:
git is having trouble seeing that I just put the old tests within this describe '.do' block - nothing changed in here